With a release build, the ALSA output plugin writes noisy messages to the log
file. Only make it noisy in a debug build.
--- a/src/output/alsa-playback/alsaaudio.cpp
+++ b/src/output/alsa-playback/alsaaudio.cpp
@@ -27,7 +27,12 @@
  ***************************************************************************/
 
 #include "alsaaudio.h"
+#ifdef QT_NO_DEBUG
+#include <QDebug>
+#define Q_DEBUG_BLOCK qDebug()
+#else
 #include "logger.h"
+#endif
 #include <qendian.h>
 
 //no debug
